README.md

```diff
@@ -1971,30 +1971,30 @@ timeline
     * Operational excellence-run/monitor system/assets=>increase business value through RA/mitigation
     * Security-protect=>increase business value through RA/mitigation
     * Reliability-recover, dynamically allocate, mitigate despite misconfiguration/problem(s)
-    * Performance efficiency-meet system requirements and maintenance through changes
+    * Performance efficiency system requirements and maintenance through changes
```

Comment:
@amirkhonov 
Let's leave in the - here to separate the pillar from the definition.  Otherwise everything looks awesome.  
Thanks for cleaning this up!
@LongBu

Reply:
@LongBu, I reverted this line. Thank you!